package.json

```diff
@@ -660,6 +667,10 @@
 					"command": "cosmosDB.loadMore",
 					"when": "never"
 				},
+				{
+					"command": "cosmosDB.api.getConnectionString",
+					"when": "never"
+				},
```

Comment:
Hide getDatabase from command palette

Reply:
Done that on the [previous PR](https://github.com/Microsoft/vscode-cosmosdb/pull/840)